modules/vba_documents/spec/mailers/monthly_report_mailer_spec.rb

```diff
@@ -25,4 +26,52 @@
   it 'sends monthly in the subject' do
     expect(@email.subject).to match(/Monthly.*/)
   end
+
+  describe '.fetch_recipients' do
+    let(:recipients) { get_fixture_yml("#{base_fixture_path}/report_recipients.yml") }
+    let(:slack_alert_email) { Settings.vba_documents.slack.default_alert_email }
+
+    context 'when environment is prod' do
+      let(:expected_result) do
+        [
+          'road.runner@va.gov',
+          'tweety.bird@va.gov',
+          'daffy.duck@va.gov',
+          'foghorn.leghorn@va.gov',
+          slack_alert_email
+        ]
+      end
+
+      before do
+        expect(VBADocuments::Deployment).to receive(:environment).and_return('prod')
+        expect(YAML).to receive(:load_file).and_return(recipients)
+        Settings.vba_documents.slack.enabled = true
+      end
+
+      it 'returns the recipients list for prod + all environments + the Slack alert email' do
+        expect(described_class.fetch_recipients.sort).to eql(expected_result.sort)
+      end
+    end
+
+    context 'when environment is staging' do
+      let(:expected_result) do
+        %w[
+          road.runner@va.gov
+          tweety.bird@va.gov
+          bugs.bunny@va.gov
+          sylvester.cat@va.gov
+        ]
+      end
```

Comment:
What happens when the env is `dev` which has no recipients for that env?

Reply:
Good call, I should add a test for that. `dev` would return the common recipients only.

Reply:
These have been added!